examples/class-async/src/lib.rs

```diff
@@ -2,29 +2,8 @@ use std::time::Duration;
 
 use fluvio_future::timer::sleep;
 
-use node_bindgen::sys::napi_value;
-use node_bindgen::core::NjError;
-use node_bindgen::core::val::JsObject;
-use node_bindgen::core::val::JsEnv;
-use node_bindgen::core::TryIntoJs;
 use node_bindgen::derive::node_bindgen;
 
-struct MyJson {
```

Comment:
can you make this `pub`, I think that it will make clippy happier and we will still be testing that it compiles

Reply:
I pushed that change